nengo_loihi/builder/inputs.py

```diff
@@ -85,3 +91,116 @@ class PESModulatoryTarget:
     def __init__(self, target):
         super().__init__()
         self.error_target = target
+
+
+class DVSFileChipNode(ChipReceiveNeurons):
+    """Node for DVS input to Loihi chip from a pre-recorded file
+
+    Parameters
+    ----------
+    file_path : string
+        The path of the file to read from. Can be a ``.aedat`` or ``.events`` file.
+    kind : "aedat" or "events" or None, optional
+        The format of the file. If ``None`` (default), this will be detected from the
+        file extension.
+    t_start : float, optional
+        Offset for the time in the file to start at, in seconds.
+    rel_time : bool, optional
+        Whether to make all times relative to the first event, or not. Defaults
+        to True for ``.aedat`` files and False otherwise.
+    pool : (int, int), optional
+        Number of pixels to pool over in the vertical and horizontal
+        directions, respectively.
+    channels_last : bool, optional
+        Whether to make the channels (i.e. the polarity) the least-significant
+        index (True) or the most-significant index (False).
+    """
+
+    def __init__(
+        self,
+        file_path,
+        format=None,
+        t_start=0,
+        rel_time=None,
+        pool=(1, 1),
+        channels_last=True,
+        label=None,
+    ):
+        self.file_path = file_path
+        assert os.path.exists(self.file_path)
+        self.format = format
+        self.t_start = t_start
+        self.rel_time = rel_time
+
+        self.dvs_height = 180
+        self.dvs_width = 240
```

Comment:
So these are standard for the DVS/Davis sensor. But one user was asking about different sizes. We might as well make them configurable; we just want to make it clear that you should only be changing it if you actually have a sensor/file format that is a different size (i.e., not just because you want e.g. a smaller image for your model). I'm also not sure if NxSDK supports live DVS in other sizes (my impression from before was that they didn't, but that could have changed or I could just be wrong), but I'm OK with there being some features on the file DVS that aren't supported on live DVS.